### PR TITLE
Gathering detail plots into directories, online & offline & relval & …

### PR DESCRIPTION
**Pull request by quark2**
#### PR description:
We have several updates on GEM onlineDQM.

- Plots for detailed information per chambers are gathered into directories
- Online & offline & relval & allplot modes
- Removed non-useful plots and stuffs

#### PR validation:
Test are done and one can check again by `runTheMatrix` workflows

@jshlee @watson-ij

### CODE REVIEW

**Comment by jshlee** [comment]
please test

**Comment by jshlee** [comment]
please test

File: DQM/GEM/src/GEMDQMBase.cc [hunk 1]
```diff
@@ -224,3 +237,124 @@ int GEMDQMBase::GenerateMEPerChamber(DQMStore::IBooker& ibooker) {
...
+
+  stationInfo.fMinPhi_ = 1048576.0;
+  for (auto p : listDivPhi) {
+    if (p.first == 1) {
+      stationInfo.fMinPhi_ = p.second.first.first;
+      break;
+    }
+  }
+
+  // For a future usage
```
Comment (civanch) [inline DQM/GEM/src/GEMDQMBase.cc:353]: No sense keeping these commented lines

**Comment by emanueleusai** [comment]
+1

**Comment by civanch** [comment]
+1

File: DQM/GEM/src/GEMDQMBase.cc (hunk 1 above)
Comment (qliphy) [reply DQM/GEM/src/GEMDQMBase.cc:353]: > No sense keeping these commented lines

@quark2 Please address above comments.

Comment (quark2) [reply DQM/GEM/src/GEMDQMBase.cc:353]: Hi @civanch and @qliphy, 

These lines are kept for future extensions. It's in printing-out format but if interfaces for this information are prepared, then these lines will be revived within the interface. This is the purpose of keeping these lines.

**Comment by qliphy** [comment]
@quark2  Please check the comparison differences are as expected or not?
https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_12_4_X_2022-05-04-1100+703215/50056/validateJR.html
Moreover, are we expecting to see GEM DQM differences?

**Comment by quark2** [comment]
Hi @qliphy, 

All the differences come from `edmErrorSummaryEntrys_logErrorHarvester__RECO`, and I think it's not related to the current update on GEM DQM.

And we don't expect a significant change in the existing plots in offlineDQM.

Best regards, 
Byeonghak Ko

**Comment by qliphy** [comment]
+1

**Pull request merged**